### DISCUSSION TITLE
Release Cadence

### DISCUSSION THREAD

**Discussion by qjones7** [keycloak/keycloak#19916] [Q&A]
I was looking into the channels and I got confused with the names of the channels ( alpha, candidate, and fast). Traditionally, 
those channels are not stable/prod channels. Is it possible to get clear definitions of the channels, what is the Release Cadence and where are the new releases broadcasted?

**Comment by vmuzikar** [comment]
Thanks for the question. If you are referring to the Operator release channels, the only currently active channel is the default one: `fast`. That [means](https://olm.operatorframework.io/docs/best-practices/channel-naming/):
>Released, supported operators which are still being monitored to assess stability/quality prior to promoting them as stable.

The other channels are kept there for historical reasons as an archive of old Operator releases. I hope that makes sense.

**Comment by qjones7** [comment]
thanks for your reply! In the doc link you sent, the names of operator
release channels are listed as Stable, Candidate, and Fast. So does alpha
mean stable? Is there a release cadence?

**Reply by vmuzikar** [reply]
`alpha` was used only by our deprecated old Operator (no longer released) and the channel name predates current naming conventions. I'd suggest considering only `fast` channel.

Releases for the Operator are aligned 1:1 with Keycloak.
